wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearningSetup/ActiveLearningInitialLabels.vue

```diff
@@ -174,10 +178,10 @@ export default Vue.extend({
                 if (categoryIndex !== 0) {
                     // For each non-default category, get all the labeled indices
                     // for this superpixel element.
-                    const labeledSuperpixels = [];
+                    const labeledSuperpixels = new Set();
```

Comment:
Switched from arrays to Sets as the data structure. Our main operations here are adding a deleting, and it wouldn't make sense for a superpixel to be labeled twice, so I think this is the appropriate data structure.